libact/query_strategies/tests/test_realdata.py

```diff
@@ -0,0 +1,112 @@
+"""
+This module uses real world dataset to test the active learning algorithms.
+Since creating own artificial dataset for test is too time comsuming so we would
+use real world data, fix the random seed, and compare the query sequence each
+active learning algorithm produces.
+"""
+import random
+import os
+import unittest
+
+from numpy.testing import assert_array_equal
+import numpy as np
+
+from libact.base.dataset import Dataset, import_libsvm_sparse
+from libact.models import LogisticRegression
+from libact.query_strategies import *
+
+
+def run_qs(trn_ds, qs, truth, quota):
+    ret = []
+    for _ in range(quota):
+        ask_id = qs.make_query()
+        trn_ds.update(ask_id, truth[ask_id])
+
+        ret.append(ask_id)
+    return np.array(ret)
+
+class RealdataTestCase(unittest.TestCase):
+
+    def setUp(self):
+        dataset_filepath = os.path.join(
+            os.path.dirname(os.path.realpath(__file__)), 'datasets/heart_scale')
+        self.X, self.y = import_libsvm_sparse(dataset_filepath).format_sklearn()
+        self.quota = 10
+
+    def test_quire(self):
+        np.random.seed(1126)
+        trn_ds = Dataset(self.X,
+                         np.concatenate([self.y[:5], [None]*(len(self.y)-5)]))
+        qs = QUIRE(trn_ds)
+        qseq = run_qs(trn_ds, qs, self.y, self.quota)
+        assert_array_equal(
+            qseq, np.array([117, 175, 256, 64, 103, 118, 180, 159, 129, 235]))
+
+    def test_RandomSampling(self):
+        random.seed(1126)
```

Comment:
I think np.random.seed(1126) and ranom.seed(1126) have the same effect? Do we want to unify the usage?